### PR TITLE
PXB-3291 Remove offensive terminology from 8.4 docs (8.4)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/how-xtrabackup-works.md
	modified:   docs/index.md
	modified:   docs/make-backup-in-replication-env.md
	modified:   docs/quickstart-overview.md
	deleted:    docs/release-notes/8.1.0-1.md
	deleted:    docs/release-notes/8.2.0-1.md
	deleted:    docs/release-notes/8.3.0-1.md
	modified:   docs/release-notes/release-notes.md
	modified:   docs/set-up-replication.md
	modified:   docs/xtrabackup-files.md
	modified:   docs/xtrabackup-option-reference.md
	modified:   variables.yml
modified:   docs/flush-tables-with-read-lock.md

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]

File: docs/set-up-replication.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ We use this backup tool. Install Percona XtraBackup on both computers for conven
 
 !!! note
    
-    It is not recommended to mix MySQL variants (Percona Server, MySQL) in your replication setup. This may produce incorrect `xtrabackup_slave_info` file when adding a new replica.
```
Comment (satya-bodapati) [inline docs/set-up-replication.md]: xtrabackup_slave_info is still there. This means, the offensive terminology file exists and we use it :( Please keep this as is.

File: docs/set-up-replication.md [hunk 2]
```diff
@@ -254,11 +254,11 @@ At the `Replica`, do a full backup:
...
 ```{.bash data-prompt="$"}
 $ xtrabackup --user=yourDBuser --password=MaGiCiGaM \
-   --backup --slave-info --target-dir=/path/to/backupdir
+   --backup --replica-info --target-dir=/path/to/backupdir
 ```
 
-By using the `--slave-info` *Percona XtraBackup* creates additional file
-called `xtrabackup_slave_info`.
+By using the `--replica-info` *Percona XtraBackup* creates additional file
+called `xtrabackup_replica_info`.
```
Comment (satya-bodapati) [inline docs/set-up-replication.md]: same here, --slave-info option still exists :(

File: docs/set-up-replication.md [hunk 3]
```diff
@@ -298,14 +298,14 @@ Make sure you change the server-id variable in `/etc/mysql/my.cnf` to 3 and
...
-skip-slave-start
+skip-replica-start
 server-id=3
 ```
 
 After setting `server_id`, start mysqld.
 
 Fetch the source_log_file and source_log_pos from the
-file `xtrabackup_slave_info`, execute the statement for setting up the
+file `xtrabackup_replica_info`, execute the statement for setting up the
```
Comment (satya-bodapati) [inline docs/set-up-replication.md]: revert to slave_info

Comment (satya-bodapati) [reply docs/set-up-replication.md]: here and everywhere?

**Review by satya-bodapati — CHANGES_REQUESTED** [review]
xtrabackup options are not changed yet. Please revert them

File: docs/xtrabackup-option-reference.md [hunk 4]
```diff
@@ -843,20 +843,20 @@ You cannot use this option with [`--stream`](#stream).
...
+This option is implemented to deal with [replication and temporary tables](https://dev.mysql.com/doc/refman/{{vers}}/en/replication-features-temptables.html) and isn’t necessary with row-based replication.
 
 Using a safe-slave-backup option stops the SQL replica thread before copying the InnoDB files.
 
 ### safe-slave-backup-timeout
 
 Usage: `--safe-slave-backup-timeout=SECONDS`
 
-How many seconds the `--safe-slave-backup` option waits for the `Slave_open_temp_tables` to become zero. The default value is 300 seconds.
+How many seconds the `--safe-replic-backup` option waits for the `Replic_open_temp_tables` to become zero. The default value is 300 seconds.
```
Comment (satya-bodapati) [inline docs/xtrabackup-option-reference.md]: safe-slave-backup instead of --safe-replic-backup

Comment (satya-bodapati) [inline docs/xtrabackup-option-reference.md]: Replic/Replica

**Review by satya-bodapati — CHANGES_REQUESTED** [review]

File: docs/set-up-replication.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/set-up-replication.md]: reverted

File: docs/set-up-replication.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/set-up-replication.md]: reverted

File: docs/set-up-replication.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/set-up-replication.md]: reverted

File: docs/xtrabackup-option-reference.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/xtrabackup-option-reference.md]: reverted

File: docs/xtrabackup-option-reference.md [hunk 5]
```diff
@@ -843,20 +843,20 @@ You cannot use this option with [`--stream`](#stream).
...
+This option is implemented to deal with [replication and temporary tables](https://dev.mysql.com/doc/refman/{{vers}}/en/replication-features-temptables.html) and isn’t necessary with row-based replication.
 
 Using a safe-slave-backup option stops the SQL replica thread before copying the InnoDB files.
 
 ### safe-slave-backup-timeout
 
 Usage: `--safe-slave-backup-timeout=SECONDS`
 
-How many seconds the `--safe-slave-backup` option waits for the `Slave_open_temp_tables` to become zero. The default value is 300 seconds.
+How many seconds the `--safe-slave-backup` option waits for the `Replic_open_temp_tables` to become zero. The default value is 300 seconds.
```
Comment (satya-bodapati) [inline docs/xtrabackup-option-reference.md]: typo?: a missing in Replica

Comment (alina-derkach-oaza) [reply docs/xtrabackup-option-reference.md]: fixed

**Review by satya-bodapati — APPROVED** [review]
LGTM!

**Pull request merged**